.travis.yml

```diff
@@ -22,12 +22,11 @@ before_install:
   - google-chrome-stable --headless --disable-gpu --no-sandbox --remote-debugging-port=9222 http://localhost &
 
 before_script:
-  - bin/rails db:test:prepare
   - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
   - chmod +x ./cc-test-reporter
   - ./cc-test-reporter before-build
 
-script: bundle exec rake ci
+script: RAILS_ENV=test bundle exec rake ci
```

Comment:
Do you need the explicit `RAILS_ENV=test` on this line, given it's in the `env.global` block below on https://github.com/sul-dlss/argo/pull/1300/files#diff-354f30a63fb0907d4ad57269548329e3R37?

Reply:
Sorry, I thought I deleted that env.global.